index.html

```diff
@@ -24,9 +24,14 @@
           </div>
         </a>
         <div class="menu">
-          <a href="#">
+          <a href="#" class="hamburger-menu">
             <img src="src/icons/hamburger.svg" alt="">
           </a>
+          <ul class="desktop-menu">
+            <li><a href="#">Portfolio</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact</a></li>
```

Comment:
[OPTIONAL] This is optional but recommended when someone clicks on any link its scrolls down to a specific section like the `work` section, `about` section and `contact` section please implement this functionality and check my suggestion below :point_down: 

```suggestion
            <li><a href="#work">Portfolio</a></li>
            <li><a href="#about">About</a></li>
            <li><a href="#contact">Contact</a></li>
```

- Add `id` to all the sections like `<section class='works' id='work'>`, `<section class="about-myself" id='about'>` and `<section class="contact-section" id='contact'>`